code/game/machinery/telecomms/broadcasting.dm

```diff
@@ -181,6 +181,17 @@
 			radio.last_chatter_time = world.time
 	//WS edit end
 
+//for all the widebands that heard the message, log it in their voice log
+	for(var/obj/item/radio/wideband in radios)
+		if(wideband.frequency == FREQ_WIDEBAND)
+			var/name = data["name"]
+			var/list/log_details = list()
+			log_details["name"] = "[name]▸"
+			log_details["message"] = "\"[html_decode(message)]\""
+			log_details["time"] = station_time_timestamp()
+			wideband.loglist.Insert(1, list(log_details))
+			wideband.log_trim()
+
```

Comment:
```suggestion
		if(radio.log)
			var/name = data["name"]
			var/list/log_details = list()
			log_details["name"] = "[name]▸"
			log_details["message"] = "\"[html_decode(message)]\""
			log_details["time"] = station_time_timestamp()
			radio.loglist.Insert(1, list(log_details))
			radio.log_trim()

```
I'd rather it just log if the option is enabled, we can maybe add specific logging intercomms or radios at some point, dunno. Plus combining it into one loop's more efficient anywho

Reply:
i trust that your code works better than my ctrl+c ctrl+v